src/ros_filter.cpp

```diff
@@ -887,7 +887,10 @@ namespace RobotLocalization
 
     // Check if the filter should start or not
     nhLocal_.param("disabled_at_startup", disabledAtStartup_, false);
-    if (!disabledAtStartup_) enabled_ = true;
+    if (!disabledAtStartup_)
```

Comment:
Actually, sorry, should have pointed this out before, but consider just doing `enabled_ = !disabledAtStartup_`.